Yafc.Model/Serialization/PropertySerializers.cs

```diff
@@ -70,14 +70,15 @@ internal sealed class ValuePropertySerializer<TOwner, TPropertyType>(PropertyInf
 
     private void setter(TOwner owner, TPropertyType? value)
         // TODO (yafc-ce/issues/256): unwrap this one-liner
-        => _setter(owner ?? throw new ArgumentNullException(nameof(owner)),
-            value ?? (CanBeNull ? default :
-            throw new InvalidOperationException($"{property.DeclaringType}.{propertyName} must not be set to null.")));
+        => _setter(owner ?? throw new ArgumentNullException(nameof(owner)), value ?? (CanBeNull
```

Comment:
This part was a revert from one of the previous wraps. It looked jagged because it was wrapped when the limit was 120 characters, so I had to use many lines. Now with 190 character limit, I applied a different wrapping. 

I thought it looked fine, but if you consider the result not good, then I'm open to alternatives.

Reply:
Yeah the old split was even worse, but i dont like the new situation where the ternary starts at the end of the line and the double use of `??` before it. I had to look at it 3x before i understood that the `throw new Argument` part was broken up by the comma.

So that adds a newline after the comma, and with that, it makes sense to give the first argument its own line too. (I was referring to the first instance of `owner`).

Reply:
Overall, this whole expression begs to be refactored into something more understandable. I just haven't done it in this PR. We can re-wrap it as you desire in the meantime.

Reply:
Turns out there's already a `TODO` above this expression that says to refactor it.
I added a bit more wrapping in https://github.com/shpaass/yafc-ce/pull/270/commits/9564e9222f0819922923307ced2e07089e2f9fad. Please resolve this discussion if you consider the point addressed.

Reply:
Yes https://github.com/shpaass/yafc-ce/commit/9564e9222f0819922923307ced2e07089e2f9fad looks great. I'm not sure why i'm not seeing that change here in the pr?

Also shouldn't line 74-75 have 1 more level of indentation?

Reply:
If by "here" you mean this conversation, it's because the diff in this conversation became outdated. Regarding indentation, I'm tired of fixing this expression, but fine xD
gimme a sec